### DISCUSSION TITLE
Osu can't import any old songs

### DISCUSSION THREAD

**Discussion by UnknownCoke** [ppy/osu#23633] [Q&A]
### Type

Crash to desktop

### Bug description

When I use the setup wizard, it automatically kicks me out of the game. When I also click go to beatmaps and no maps are showing, it also kicks me out of the game

### Screenshots or videos

I can't send a recording since my file is too big (I tried using a compressor too) and it is too complicated to be sent with pictures.  I could send it over discord or email. Discord is UnknownCoke#0793

### Version

2023.513.0

### Logs

Logs aren't accessible on mac. File can't be found
[message.txt](https://github.com/ppy/osu/files/11537642/message.txt)

**Comment by UnknownCoke** [comment]
P.s. It is not the entirety of the error code. It has about 1k lines of code and idk to post that

**Comment by peppy** [comment]
Unfortunately the log you posted is truncated and therefore contains no useful information.

But we also need `runtime.log`.
